### DISCUSSION TITLE
Name and Description - OON object

### DISCUSSION THREAD

**Discussion by lyaoPBC** [CMSgov/price-transparency-guide#248] [Q&A]
What is the difference for name and description fields?  are they both just the description of billing code?  Thanks

**Comment by APWU-GRoberts** [comment]
We are using a combination of name = short description and description = long description for these fields. 

Example:
billing_code 97110
billing_code_name THERAPEUTIC EXERCISES
billing_code_description THERAPEUTIC PX 1/> AREAS EACH 15 MIN EXERCISES

**Comment by Kvillafor** [comment]
@shaselton Can you elaborate on how the Name and Description fields should be reported on the Allowed Amounts file?  For instance, @apwu-groberts above is using the short description for the Name field and a long description for the Description field.  

1. Are there any specific guidelines for the Name field, since CMS recommends using the short description in the Description field?
2. Is there a standard short description for the different healthcare codes? If so, where would we be able to find these?  
3. Can we use the same description for the Name and Description field?  There are no character limits in the schema that CMS provided.

We currently only store a long description for all healthcare codes and need clarification on whether we are required to report the short description or not.

**Reply by tdfow** [reply]
@Kvillafor we are populating with the same value, as our customers only provided one name/description field, so we will be populating with same value.
